src/app/map/components/map-controls/zoom-controls/zoom-controls.component.ts

```diff
@@ -62,7 +71,19 @@ export class ZoomControlsComponent implements OnInit, OnDestroy {
   private initSubscriptions() {
     this.subscriptions.add(this.isMaxZoomedIn$.pipe(tap((value) => (this.isMaxZoomedIn = value))).subscribe());
     this.subscriptions.add(this.isMaxZoomedOut$.pipe(tap((value) => (this.isMaxZoomedOut = value))).subscribe());
-    this.subscriptions.add(this.geolocationState$.pipe(tap((value) => (this.geolocationState = value))).subscribe());
+    this.subscriptions.add(
+      this.geolocationState$
+        .pipe(
+          tap((value) => {
+            this.geolocationState = value;
+            this.locationButtonTooltipText =
```

Comment:
Is this a new requirement to add the error to the tooltip?

Reply:
No, i just moved it from the html into the component due to the Constans. Otherwise I would have had to have the same logic fotwice for the label and the tooltip